packages/alfa-rules/test/sia-er62/common.ts

```diff
@@ -22,6 +22,13 @@ export function addOutline(
   );
 }
 
+export function getContainerColor(color: RGB): Contrast.Color {
+  return Contrast.Color.of("container", color);
+}
+
+export function getLinkColor(color: RGB): Contrast.Color {
+  return Contrast.Color.of("link", color);
+}
```

Comment:
Since these are always used together, it might be worth doing a 
```typescript
function makePairing(container, link, contrast) {
return Pairing.of(Color.of("container", container), Color.of("link", link), contrast)
}
```
Will make call sites more compact.

(or it calls the new `Pairing.from(["container", container], ["link", link], contrast`)